{"name": "I double click the double click button", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"css selector\",\"selector\":\"[id=\"doubleClickMessage\"]\"}\n  (Session info: chrome=122.0.6261.129); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nStacktrace:\n0   chromedriver                        0x0000000101200524 chromedriver + 3966244\n1   chromedriver                        0x00000001011f8ab0 chromedriver + 3934896\n2   chromedriver                        0x0000000100e7bda0 chromedriver + 277920\n3   chromedriver                        0x0000000100ebe394 chromedriver + 549780\n4   chromedriver                        0x0000000100ef6bf0 chromedriver + 781296\n5   chromedriver                        0x0000000100eb2fb0 chromedriver + 503728\n6   chromedriver                        0x0000000100eb3a28 chromedriver + 506408\n7   chromedriver                        0x00000001011c570c chromedriver + 3725068\n8   chromedriver                        0x00000001011c9c00 chromedriver + 3742720\n9   chromedriver                        0x00000001011ae1f4 chromedriver + 3629556\n10  chromedriver                        0x00000001011ca6fc chromedriver + 3745532\n11  chromedriver                        0x00000001011a156c chromedriver + 3577196\n12  chromedriver                        0x00000001011e8f5c chromedriver + 3870556\n13  chromedriver                        0x00000001011e9100 chromedriver + 3870976\n14  chromedriver                        0x00000001011f8720 chromedriver + 3933984\n15  libsystem_pthread.dylib             0x000000018ab46f94 _pthread_start + 136\n16  libsystem_pthread.dylib             0x000000018ab41d34 thread_start + 8\n\n", "trace": "  File \"/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"steps/buttons_steps.py\", line 16, in step_impl\n    context.buttons_page.is_double_click_msg_displayed()\n  File \"/Users/olaru/Projects/ProiectFinal/pages/buttons_page.py\", line 37, in is_double_click_msg_displayed\n    self.is_element_displayed(self.SUCCESS_MSG_DOUBLE_CLICK)\n  File \"/Users/olaru/Projects/ProiectFinal/pages/base_page.py\", line 39, in is_element_displayed\n    assert self.find(element).is_displayed(), \"Element not displayed\"\n           ^^^^^^^^^^^^^^^^^^\n  File \"/Users/olaru/Projects/ProiectFinal/pages/base_page.py\", line 13, in find\n    return self.driver.find_element(*locator)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 742, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\"using\": by, \"value\": value})[\"value\"]\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 348, in execute\n    self.error_handler.check_response(response)\n  File \"/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/selenium/webdriver/remote/errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "steps": [{"name": "Given I am on the Buttons page", "status": "passed", "start": 1710952673466, "stop": 1710952673783}, {"name": "When I double click on the double click button", "status": "passed", "start": 1710952673784, "stop": 1710952674075}, {"name": "Then Double click success message pops up", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"css selector\",\"selector\":\"[id=\"doubleClickMessage\"]\"}\n  (Session info: chrome=122.0.6261.129); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nStacktrace:\n0   chromedriver                        0x0000000101200524 chromedriver + 3966244\n1   chromedriver                        0x00000001011f8ab0 chromedriver + 3934896\n2   chromedriver                        0x0000000100e7bda0 chromedriver + 277920\n3   chromedriver                        0x0000000100ebe394 chromedriver + 549780\n4   chromedriver                        0x0000000100ef6bf0 chromedriver + 781296\n5   chromedriver                        0x0000000100eb2fb0 chromedriver + 503728\n6   chromedriver                        0x0000000100eb3a28 chromedriver + 506408\n7   chromedriver                        0x00000001011c570c chromedriver + 3725068\n8   chromedriver                        0x00000001011c9c00 chromedriver + 3742720\n9   chromedriver                        0x00000001011ae1f4 chromedriver + 3629556\n10  chromedriver                        0x00000001011ca6fc chromedriver + 3745532\n11  chromedriver                        0x00000001011a156c chromedriver + 3577196\n12  chromedriver                        0x00000001011e8f5c chromedriver + 3870556\n13  chromedriver                        0x00000001011e9100 chromedriver + 3870976\n14  chromedriver                        0x00000001011f8720 chromedriver + 3933984\n15  libsystem_pthread.dylib             0x000000018ab46f94 _pthread_start + 136\n16  libsystem_pthread.dylib             0x000000018ab41d34 thread_start + 8\n\n", "trace": "  File \"/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"steps/buttons_steps.py\", line 16, in step_impl\n    context.buttons_page.is_double_click_msg_displayed()\n  File \"/Users/olaru/Projects/ProiectFinal/pages/buttons_page.py\", line 37, in is_double_click_msg_displayed\n    self.is_element_displayed(self.SUCCESS_MSG_DOUBLE_CLICK)\n  File \"/Users/olaru/Projects/ProiectFinal/pages/base_page.py\", line 39, in is_element_displayed\n    assert self.find(element).is_displayed(), \"Element not displayed\"\n           ^^^^^^^^^^^^^^^^^^\n  File \"/Users/olaru/Projects/ProiectFinal/pages/base_page.py\", line 13, in find\n    return self.driver.find_element(*locator)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 742, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\"using\": by, \"value\": value})[\"value\"]\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 348, in execute\n    self.error_handler.check_response(response)\n  File \"/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/selenium/webdriver/remote/errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "start": 1710952674075, "stop": 1710952677104}, {"name": "And The message is \"You have done a double click\"", "status": "skipped", "start": 1710952677107, "stop": 1710952677107}], "start": 1710952673466, "stop": 1710952677107, "uuid": "b47388b6-0457-4ffc-aa61-8e2a7d0bcf7c", "historyId": "6b7ed6b84d9fc3f0818435cc2c254ad4", "fullName": "Buttons: I double click the double click button", "labels": [{"name": "severity", "value": "normal"}, {"name": "tag", "value": "ButtonsPage"}, {"name": "tag", "value": "sanity"}, {"name": "feature", "value": "Buttons"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}